C/plugins/utils/get_plugin_info.cpp

```diff
@@ -41,22 +41,23 @@ int main(int argc, char *argv[])
     exit(1);
   }
 
+  openlog("Fledge PluginInfo", LOG_PID|LOG_CONS, LOG_USER);
```

Comment:
Better If we can avoid to use `Product prefix`

Reply:
I disagree here, much better to identify it is part of t Fledge core, in the same way we have Fledge Storage